engineapi/backups.go

```diff
@@ -40,24 +42,46 @@ func NewBackupTarget(backupTarget, engineImage string, credential map[string]str
 	}
 }
 
-func (b *BackupTarget) LonghornEngineBinary() string {
-	return filepath.Join(types.GetEngineBinaryDirectoryOnHostForImage(b.Image), "longhorn")
+// getBackupCredentialEnv returns the environment variables as KEY=VALUE in string slice
+func getBackupCredentialEnv(backupTarget string, credential map[string]string) ([]string, error) {
```

Comment:
Put this modification in the 2nd commit. Or directly coalesce 2 commits. It's confusing to add this function in this commit.

Reply:
I removed the enhancement code out in this PR to make it easier to review.